.github/workflows/ci.yaml

```diff
@@ -0,0 +1,27 @@
+name: ppp github action ci 
+
+on: 
+  push:
+    branches:
+      - master
+  pull_request:
```

Comment:
@geoffrey1330 since we're going to push updates to master branch, this job shouldn't run on pull requests.

Reply:
@anuveyatsu Alright i will remove the pull request trigger

Reply:
@anuveyatsu here is the new commit 
https://github.com/datasets/ppp/pull/12/commits/2d7456d4a71fb947e316e2677a1e9e8981d890b3